data/libwacom.stylus

```diff
@@ -749,3 +749,22 @@ Type=Puck
 HasLens=false
 HasWheel=true
 Buttons=5
+
+[0x10002]
+# Movink
+Name=UD Pen
+Group=udpen
+PairedStylusIds=0x1000a;
+Buttons=2
+EraserType=Invert;
+Axes=Tilt;Pressure;Distance;
+Type=General
+
+[0x1000a]
+# Movink
+Name=UD Pen Eraser
+Group=udpen
+PairedStylusIds=0x10002;
+EraserType=Invert;
+Axes=Tilt;Pressure;Distance;
+Type=General
```

Comment:
Missing a `Buttons` line. Looks like the old pens all had 1 button for the eraser but I  no longer remember why this was the case... I think this predates the distinction between tip and buttons so `Buttons=1` is necessary here.